### PR TITLE
Style & psr fixes 

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
  */
 class <?php echo $testClass ?> extends \PHPUnit\Framework\TestCase implements EndToEndInterface {
 
-  public static function setUpBeforeClass() {
+  static public function setUpBeforeClass() {
```
Comment (seamuslee001) [inline src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php:33]: My understanding is visibility first

Comment (eileenmcnaughton) [reply src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php:33]: IDE seems fairly willing to back up it's claim...

Screen Shot 2020-07-03 at 11 07 56 AM

Comment (eileenmcnaughton) [reply src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php:33]: hang on you are right...

Comment (eileenmcnaughton) [reply src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php:33]: I reversed the wrong ones

Comment (eileenmcnaughton) [reply src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php]: fixed

**Comment by totten** [comment]
This looks pretty good. I'm just gonna run `tests/make-example.sh` before merging.

File: src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php [hunk 2]
```diff
@@ -144,17 +154,19 @@ public function executeSqlTemplate($tplFile) {
...
    *
    * This is just a wrapper for CRM_Core_DAO::executeSql, but it
-   * provides syntatic sugar for queueing several tasks that
+   * provides syntactic sugar for queueing several tasks that
    * run different queries
+   *
+   * @return bool
    */
-  public function executeSql($query, $params = array()) {
+  public function executeSql($query, $params = []]) {
```
Comment (totten) [inline src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php:162]: @eileenmcnaughton while running `make-example.sh`, it aborted here:

```
PHP Parse error:  syntax error, unexpected ']', expecting ')'
in /Users/totten/bknix/build/dmaster/web/sites/all/modules/civicrm/tools/extensions/
org.civicrm.civixexample/CRM/Civixexample/Upgrader/Base.php on line 159
```

File: src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php [hunk 3]
```diff
@@ -100,7 +106,7 @@ public function executeCustomDataFile($relativePath) {
    *
    * @return bool
    */
-  protected static function executeCustomDataFileByAbsPath($xml_file) {
+  protected function executeCustomDataFileByAbsPath($xml_file) {
```
Comment (totten) [inline src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php:109]: Oh, yeah, the call to `$this->executeCustomDataFileByAbsPath()` doesn't match the official signature. So this fixes the signature.

I grepped `universe` to see if the signature change might be a problem -- fortunately, almost nobody else calls the function directly; and the ones who do are using the `$this->` formulation.

So looks good. :+1:

File: src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php (hunk 2 above)
Comment (totten) [reply src/CRM/CivixBundle/Resources/views/Code/upgrader-base.php.php]: Cool, looks good now.

**Pull request merged**